.github/workflows/release_portable_linux_packages.yml

```diff
@@ -322,3 +322,39 @@ jobs:
         with:
           path: ${{ env.OUTPUT_DIR }}/caches
           key: portable-linux-package-matrix-v1-${{ matrix.target_bundle.amdgpu_family }}-${{ github.sha }}
+
+  build_native_rpm_package:
+    name : Build Native RPM Packages
```

Comment:
Let's iterate on the workflow job names a bit here. See how on https://github.com/ROCm/TheRock/actions/runs/19488211687 the new names get cut off in the UI

<img width="1353" height="574" alt="Image" src="https://github.com/user-attachments/assets/e13477ae-41e2-434f-81c6-1bb9c5396c29" />

How about we match https://github.com/ROCm/TheRock/blob/efed3c3b10a5cce8578f58f8eb288582c26d18c4/.github/workflows/release_portable_linux_packages.yml#L131-L132

and call this `${{ matrix.target_bundle.amdgpu_family }}::Build Native RPM`?